dbt/include/bigquery/macros/adapters/apply_grants.sql

```diff
@@ -0,0 +1,27 @@
+{% macro  bigquery__get_show_grant_sql(relation) %}
+{# Note: This only works if the location is defined in the profile.  It is an optional field right now. #}
+{# TODO: location is hardcoded for now - need logic around this (default to us)#}
+select * from {{ relation.project }}.`region-us`.INFORMATION_SCHEMA.OBJECT_PRIVILEGES
```

Comment:
Per [docs](https://cloud.google.com/bigquery/docs/information-schema-object-privileges):

> Queries against this view must include a [region qualifier](https://cloud.google.com/bigquery/docs/information-schema-intro#syntax).

This appears to be our first such case. We document that `location` can be set to either `US` or `EU`, but it looks like it's `None` by default. Hm.

I think we have two options:
- Pick a default if not set, probably `US` :)
- Require that folks using `grants` on BigQuery set `location` in their connection profile (not a big ask tbh)
```sql
{% set location = target.get('location', 'US') | lower %}
select * from {{ relation.project }}.`region-{{ location }}`.INFORMATION_SCHEMA.OBJECT_PRIVILEGES
```